### PR TITLE
chore: [NODE-1384] Update ansible for upcoming lints [S3_UPLOAD]

### PR DESCRIPTION
**Pull request by Bownairo**
Our current `ansible-lint` is too old to be installed in the upcoming 24.04 upgrade. The new version has a lot more lints that need satisfied.

---

We'd like to remove all of this code soon. This _should_ be a noop maintenance update, until then.

### CODE REVIEW

**Comment by r-birkner** [comment]
Thanks a lot for taking on this task! With the latest changes, we can still deploy the boundary nodes :)

**Review by joeodfinity — APPROVED** [review]
The number of gratuitous changes in capitalization/quoting/non-quoting obscures the core changes, but the goal is to remove most of this support in the near term, as the ansible here is mostly for driving deployments on the legacy static testnets and the legacy (non-k8s hosted) version of farm, both of which have few days left.

File: .ansible-lint-ignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This file contains ignores rule violations for ansible-lint
```
Comment (basvandijk) [inline .ansible-lint-ignore:1]: Any chance we could move this file under `testnet/ansible`?

**Review by basvandijk — APPROVED** [review]

**Review by nabdullindfinity — APPROVED** [review]

**Review by marko-k0 — APPROVED** [review]

**Comment by Bownairo** [comment]
> The number of gratuitous changes in capitalization/quoting/non-quoting obscures the core changes

These should be the only changes. The linter itself will be upgraded along with the images, the goal is to keep that MR as small as possible.

**Pull request merged**